SPECS/libxkbcommon/libxkbcommon.spec

```diff
@@ -1,30 +1,20 @@
-%define required_wayland_version 1.2.0
-%define required_wayland_protocols_version 1.7
-
-Summary:        X.Org X11 XKB parsing library
 Name:           libxkbcommon
-Version:        1.5.0
-Release:        1%{?dist}
-License:        MIT
+Version:        1.6.0
+Release:        2%{?dist}
+Summary:        X.Org X11 XKB parsing library
+License:        MIT AND X11 AND MIT-CMU
+URL:            http://www.x.org
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
-URL:            https://www.x.org
-Source0:        https://xkbcommon.org/download/%{name}-%{version}.tar.xz
+Distribution:   Microsoft Azure Linux
 
-BuildRequires:  bison
-BuildRequires:  flex
-BuildRequires:  git
-BuildRequires:  libX11-devel
-BuildRequires:  libxml2-devel
-BuildRequires:  meson
-BuildRequires:  pkg-config
+Source0:        https://xkbcommon.org/download/%{name}-%{version}.tar.xz
+BuildRequires:  gcc
+BuildRequires:  git meson
+BuildRequires:  byacc flex bison
+BuildRequires:  xorg-x11-proto-devel libX11-devel
```

Comment:
Please use one dependency per line. Ideally also in alphabetical order but that's a P2.

Reply:
no.

this is how our inherited spec file lists it; it is counterproductive to change that for zero technical reason.